test/analysis/buildTypeWeights.test.ts

```diff
@@ -1176,6 +1258,12 @@ describe('Test buildTypeWeightsFromSchema function', () => {
             );
         });
 
+        test('there is an unbounded list and user chooses to enforce bounded lists', () => {
```

Comment:
Do we have a test where there is an unbounded list and the user chose not to enforce unbounded lists?

Reply:
I have accounted for that in the weightFunction but there is no test. I will make one.

Reply:
Although every other test implicitly checking for this.